config/sitemap.rb

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
If we want to use the rake tasks already provided by `sitemap_generator`, they're based off running the `config/sitemap.rb` file. You can change its location like so:

```
CONFIG_FILE="config/some_other_sitemap.rb" rake sitemap:refresh
```

but the code that builds the sitemaps is supposed to live in `config/sitemap.rb` by design.  See https://github.com/kjvarga/sitemap_generator/blob/master/lib/sitemap_generator/tasks.rb

We will only run the sitemaps in production so we shouldn't need any configuration outside of the `sitemap.rb` and we can put the `sitemap.rb` as a shared file rather than in github. 

Alternatively, I can implement a `sitemap_refresh` Rake task that copies their code into our own tasks. Thoughts?